APP/src/api.js

```diff
@@ -291,29 +291,41 @@ export const postTeamPictures = async (year, event, team, data, callback) => {
   try {
     const endpoint = `${API_ENDPOINT}/${year}/${event}/frc${team}/pictures/`;
     console.log(endpoint);
-
-    // Convert base64 string to Blob
     const blob = dataURItoBlob(data);
-
-    // Create FormData object
     const formData = new FormData();
-    formData.append('data', blob, 'image.jpg'); // 'data' is the key, 'image.jpg' is the filename
-
+    formData.append('data', blob, 'image.jpg');
     const response = await fetch(endpoint, {
       method: 'POST',
       body: formData,
     });
-
     callback(response.status);
   } catch (e) {
     console.error('Error:', e);
     callback(0);
   }
 };
 
+export const deleteTeamPictures = async (year, event, team, data, callback) => {
```

Comment:
This should use a DELETE method instead of /DeletePictures. If you are deleting a picture, why are you sending a picture as a part of the message?